### ISSUE TITLE
[BUG] "Person" filter button is missing an icon.

### ISSUE THREAD

**Issue by kenieh** [CDCgov/prime-simplereport#3773]
### Description
In the "People" tab, the "Person" filter is missing a "magnify glass" icon in the filter button. See the screenshot below:

![Screen Shot 2022-05-16 at 12.15.21 PM.png](https://images.zenhubusercontent.com/60da5912fad056ef8402132e/64055bf6-7abd-42ff-872e-b5f1ea709f1c)

This issue also appears on the Conduct Test page and needs to be resolved there too.

![Screen Shot 2022-05-16 at 4.08.09 PM.png](https://images.zenhubusercontent.com/60da5912fad056ef8402132e/86ca05fc-41a2-48f9-9221-ccc5fe9a0840)

### Expected behavior
There should be a "magnify glass" icon in the filter button as following the USWDS design system (https://designsystem.digital.gov/components/search/)

![Screen Shot 2022-05-16 at 3.43.08 PM.png](https://images.zenhubusercontent.com/60da5912fad056ef8402132e/9efc4b0d-51c7-4e46-808e-8e9003a303fd)

**Comment by emmastephenson** [comment]
The ask here is to change search boxes across the app (both on the conduct tests and people page) to make sure the search icon displays.

Bob ran into a similar problem on the static site and resolved through fixing the paths for the USWDS icon.

**Comment by emmastephenson** [comment]
This is also an accessibility issue, because the magnifying glass functions as a permanent label for the search box.

See https://docs.google.com/spreadsheets/d/1b6xjubAN5Uw11DV4_B5zvSu7HeDI6Z-DpV4DMcToqAY/edit#gid=981460209&range=28:28

**Comment by emmastephenson** [comment]
This is also an issue on the conduct tests page.
